fetch.bs

```diff
@@ -1517,6 +1517,7 @@ the empty string,
 "<code>style</code>",
 "<code>track</code>",
 "<code>video</code>",
+"<code>web-identity</code>",
```

Comment:
All the other names here, such as "`sharedworker`" are without hyphen. I think I'd favor local consistency, but will defer to @domenic on this.

Reply:
Yeah, probably best to stick with local consistency (no hyphen).

Reply:
Done!